metadata-models/src/main/resources/entity-registry.yml

```diff
@@ -586,6 +586,21 @@ entities:
       - forms
       - testResults
       - subTypes
+  - name: application
+    category: core
+    keyAspect: applicationKey
+    aspects:
```

Comment:
should `dataPlatformInstance` be included too?
not sure about the scope of _Application_, so just raising the question to make sure it's not overlooked

Reply:
I don't think that should be necessary based on what we've heard so far.